src/test/resources/htsjdk/tribble/AbstractFeatureReaderTest/test.vcf

```diff
@@ -0,0 +1,24 @@
+##fileformat=VCFv4.1
```

Comment:
Can you give this file a better/more specific name than "test.vcf", since it's only used in the redirect test (or put it in a subdirectory specific to that test)?